uptrain/constants.py

```diff
@@ -22,6 +22,7 @@ class AnnotationMethod(Enum):
 
 
 class Monitor(str, Enum):
+    ACCURACY = "accuracy"
```

Comment:
do we need this?

Reply:
It's getting used. I was thinking we can have an accuracy class where we have all accuracy measures (for classification, regression, etc) are listed.